hbase-common/src/main/java/org/apache/hadoop/hbase/io/ByteBuffAllocator.java

```diff
@@ -68,9 +68,25 @@
   // default heap allocator, it will just allocate ByteBuffers from heap but wrapped by an ByteBuff.
   public static final ByteBuffAllocator HEAP = ByteBuffAllocator.createOnHeap();
 
-  public static final String MAX_BUFFER_COUNT_KEY = "hbase.ipc.server.allocator.max.buffer.count";
+  public static final String MAX_BUFFER_COUNT_KEY = "hbase.server.allocator.max.buffer.count";
 
-  public static final String BUFFER_SIZE_KEY = "hbase.ipc.server.allocator.buffer.size";
+  public static final String BUFFER_SIZE_KEY = "hbase.server.allocator.buffer.size";
+
+  @Deprecated
+  static final String DEPRECATED_MAX_BUFFER_COUNT_KEY = "hbase.ipc.server.reservoir.initial.max";
```

Comment:
These old configs are  there in code already right? Some other place.  Also referring there?  Or you removed all those?  Sorry I did not review that jira then.  Do we by chance referring these configs in config.xml?

Reply:
We removed all of them in https://issues.apache.org/jira/browse/HBASE-21916 before.   Did a finding search about the hbase.ipc.server.reservoir.initial.max & hbase.ipc.server.reservoir.initial.buffer.size,  no other places are refering the deprecated configs now except the document. 
So here it should be OK.